api4/team_test.go

```diff
@@ -386,6 +386,28 @@ func TestGetTeamStats(t *testing.T) {
 	CheckUnauthorizedStatus(t, resp)
 }
 
+func TestGetTeamUnread(t *testing.T) {
+	th := Setup().InitBasic().InitSystemAdmin()
+	defer TearDown()
+	Client := th.Client
+
+	user := th.BasicUser
+	Client.Login(user.Email, user.Password)
+
+	_, resp := Client.GetTeamUnread(model.NewRandomString(16))
+	CheckBadRequestStatus(t, resp)
+
+	_, resp = Client.GetTeamUnread(th.BasicTeam.Id)
+	CheckForbiddenStatus(t, resp)
+
+	Client.Logout()
+	_, resp = Client.GetTeamUnread(th.BasicTeam.Id)
+	CheckUnauthorizedStatus(t, resp)
+
+	_, resp = th.SystemAdminClient.GetTeamUnread(th.BasicTeam.Id)
+	CheckNoError(t, resp)
```

Comment:
Can you also make sure that a correct TeamUnread is returned here? Need at least one case where it's not being piped to `_`